store/store.go

```diff
@@ -512,14 +530,17 @@ func (bs *BlockStore) PruneBlocks(height int64, state sm.State) (uint64, int64,
 			if err := batch.Delete(bs.dbKeyLayout.CalcBlockCommitKey(h)); err != nil {
 				return 0, -1, err
 			}
+			bs.blockCommitCache.Remove(h)
```

Comment:
Do we need to do this? If we don't, new block commits will inevitably replace old / pruned commits anyway.

Reply:
We may not need to do it, got a test failure for not doing it on block parts (because the pruning tests were relying on checking that it got removed)